### PR TITLE
PLANNER-2713 Score DRL deprecation blog

### PR DESCRIPTION
**Pull request by triceo**

### CODE REVIEW

File: optaplanner-website-root/content/blog/2022/05/26/optaplanner-deprecates-score-drl.adoc [hunk 1]
```diff
@@ -0,0 +1,40 @@
+= OptaPlanner deprecates score DRL
+triceo
+2022-03-07
```
Comment (yurloc) [inline optaplanner-website-root/content/blog/2022/05/26/optaplanner-deprecates-score-drl.adoc:3]: Please don't forget to update the date before merging the PR.

File: optaplanner-website-root/content/blog/2022/05/26/optaplanner-deprecates-score-drl.adoc [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+Rest assured that support for score DRL will not be removed in OptaPlanner 8.
+The next major version of OptaPlanner, however, will no longer support score DRL.
+
+== How to migrate?
+
+We understand that constraints are a fundamental part of your application, and that refactoring them to Constraint Streams will take time.
+To make this job as easy as possible, we have prepared a https://www.optaplanner.org/learn/drl-to-constraint-streams-migration.html[migration guide] which explains the most typical use cases and migration patterns.
+
+We believe that, while reading the guide, you will realize that the migration is fairly straight-forward in most cases.
+In return, you will get fast constraints written in Java that is ready for Quarkus and the cloud, with full IDE support and a https://www.optaplanner.org/docs/optaplanner/latest/constraint-streams/constraint-streams.html#constraintStreamsTesting[comprehensive testing framework].
```
Comment (ge0ffrey) [inline optaplanner-website-root/content/blog/2022/05/26/optaplanner-deprecates-score-drl.adoc:36]:
```suggestion
In return, you will get fast constraints written in Java that are ready for Quarkus and the cloud, with full IDE support and a https://www.optaplanner.org/docs/optaplanner/latest/constraint-streams/constraint-streams.html#constraintStreamsTesting[comprehensive testing framework].
```

File: optaplanner-website-root/content/blog/2022/05/26/optaplanner-deprecates-score-drl.adoc [hunk 3]
```diff
@@ -0,0 +1,40 @@
+= OptaPlanner deprecates score DRL
+triceo
+2022-05-26
+:page-interpolate: true
+:jbake-type: post
+:jbake-tags: constraint, constraint streams, drl, deprecation
+
+Long story short: The time to switch to https://www.optaplanner.org/docs/optaplanner/latest/constraint-streams/constraint-streams.html[Constraint Streams] is now.
```
Comment (ge0ffrey) [inline optaplanner-website-root/content/blog/2022/05/26/optaplanner-deprecates-score-drl.adoc:8]:
```suggestion
TLDR: The time to switch from scoreDRL to https://www.optaplanner.org/docs/optaplanner/latest/constraint-streams/constraint-streams.html[Constraint Streams] is now.
```

File: optaplanner-website-root/content/blog/2022/05/26/optaplanner-deprecates-score-drl.adoc [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+Here's the https://www.optaplanner.org/learn/drl-to-constraint-streams-migration.html[score DRL migration guide].
+Score DRL is not going away in OptaPlanner 8.
+
+== The rise of Constraint Streams
+
+As I was starting with OptaPlanner more than a decade ago, score DRL was there, promising easy and fast score calculation.
+It wasn't as fast as the Incremental Java calculator, but it also wasn't as hard to write while still getting better results than the Easy Java calculator.
+On the other hand, it required me to learn a new programming language, and one that didn't (and still doesn't) have the best support in IDEs.
```
Comment (ge0ffrey) [inline optaplanner-website-root/content/blog/2022/05/26/optaplanner-deprecates-score-drl.adoc:16]: Can we maybe remove this negative sentence? Or make this point in a positive way, by pointing out in the next paragraph that CS is just java (or kotlin), so has excellent IDE support (code highlighting, code completion, debugging)? 

I have an old blog with some screenshots that point that out: https://www.optaplanner.org/blog/2020/04/07/ConstraintStreams.html
